.github/workflows/os-zoo.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       image: docker.io/library/alpine:${{ matrix.tag }}
     env:
       # https://www.openwall.com/lists/musl/2022/02/16/14
-      EXTRA_CFLAGS: ${{ matrix.cc == 'clang' && '-Wno-sign-compare -Wno-switch-default' || '' }}
+      EXTRA_CFLAGS: '' #${{ matrix.cc == 'clang' && '-Wno-sign-compare -Wno-switch-default' || '' }}
```

Comment:
I'm not sure why this is needed.  It seems like the github expression should evaluate to an empty string already in any case where the compiler isn't clang.  Is this some github idiosyncrasy?